public/components/trace_analytics/components/common/shared_components/sharedComponents.scss

```diff
@@ -2,23 +2,53 @@
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
- 
+
 .ellipsis-left {
-    direction: rtl; 
-    unicode-bidi: plaintext;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
+  direction: rtl;
+  unicode-bidi: plaintext;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.full-wrapper {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.normal-wrapper {
+  display: flex;
+  flex-direction: column;
+  height: auto;
+}
+
+.grid-loading-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(255, 255, 255, 0.5);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 2;
+}
+
+.dark-mode-enabled {
+  .euiDataGrid__pagination {
+    background-color: $euiColorEmptyShade;
+
+    color: #d3dae6;
```

Comment:
may be we use this: https://github.com/opensearch-project/oui/blob/03889629e54ffa0de05f3e58d1111c21397d9f04/src/components/text/_text_color.scss#L14

Reply:
Updated pagination to follow color convention
```
.dark-mode-enabled {
  .euiDataGrid__pagination {
    background-color: $euiColorEmptyShade;

    color: $euiTextColor;

    .euiButtonEmpty {
      color: $euiTextColor;

      &:hover {
        background-color: $euiColorMediumShade;
      }
    }
  }
}
```

Reply:
<img width="1242" alt="Dark" src="https://github.com/user-attachments/assets/bcbb4eae-1712-4814-b7fd-702c46fbe61c" />
<img width="1229" alt="Light" src="https://github.com/user-attachments/assets/c2023336-af30-41df-b35d-afefac652aeb" />